.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
```

Comment:
@vlachoudis : I noticed that Jarret removes the .pyc files from getting sync'd into the github repo. Do you add these for performance purposes? Or can they be removed?